src/tiled/adjusttileindexes.cpp

```diff
@@ -264,26 +264,26 @@ AdjustTileMetaData::AdjustTileMetaData(TilesetDocument *tilesetDocument)
         QVector<ChangeTileWangId::WangIdChange> changes;
 
         // Move all WangIds to their new tiles
-        for (const WangTile &wangTile : wangSet->wangTilesByWangId()) {
-            if (Tile *fromTile = wangTile.tile()) {
-                if (Tile *newTile = adjustTile(fromTile)) {
-                    WangId fromWangId = wangSet->wangIdOfTile(newTile);
-                    WangId toWangId = wangTile.wangId();
-                    changes.append(ChangeTileWangId::WangIdChange(fromWangId, toWangId, newTile));
-                }
-            }
-        }
+//        for (const WangTile &wangTile : wangSet->wangTilesByWangId()) {
+//            if (Tile *fromTile = wangTile.tile()) {
+//                if (Tile *newTile = adjustTile(fromTile)) {
+//                    WangId fromWangId = wangSet->wangIdOfTile(newTile);
+//                    WangId toWangId = wangTile.wangId();
+//                    changes.append(ChangeTileWangId::WangIdChange(fromWangId, toWangId, newTile));
+//                }
+//            }
+//        }
```

Comment:
I don't recall doing this change, so you probably know best why this should be removed.

Reply:
It's not commented out because it should be removed, it's commented out because my change was still in progress and this code didn't compile anymore. I'll fix it up once we're done with the rest. :-)